Environmental Sensing/Radon Sensor Signoff.md

```diff
@@ -0,0 +1,75 @@
+# Radon Sensor Signoff
+
+# Function of the subsystem
+
+The purpose of the radon sensor in the Autonomous Crawl Space Inspection Robot is to identify and measure the levels of radon gas in the environment. Radon is a hazardous radioactive gas that is commonly found in crawl spaces, resulting from the decay of uranium in natural stone. By integrating a radon sensor into the robot, it can gather real-time information on the radon levels present in the crawlspace. The sensor provides readings that indicate the concentration of radon gas, enabling early detection of elevated levels. By including a radon sensor, the Autonomous Crawlspace Inspection Robot improves its ability to collect environmental data, offering valuable insights into potential health risks and enabling proactive measures to protect against the dangers of radon exposure.
+
+The Radon Eye RD200 was selected by the team due to its compact and lightweight nature, distinguishing it from other radon sensors. It maintains dependable precision and accuracy when detecting radon gas levels. This guarantees the acquisition of reliable data for evaluating potential health hazards.
+
+# Constraints:
```

Comment:
Number the constraints in the table, that way when you analyze them, it is easy to refer to the number.